src/matrix/ssss/index.ts

```diff
@@ -22,7 +22,9 @@ import type {Storage} from "../storage/idb/Storage";
 import type {Transaction} from "../storage/idb/Transaction";
 import type {KeyDescriptionData} from "./common";
 import type {Platform} from "../../platform/web/Platform.js";
-import type * as Olm from "@matrix-org/olm"
+import type * as OlmNamespace from "@matrix-org/olm"
+
+type Olm = typeof OlmNamespace;
```

Comment:
In the end I'm not sure this actually does anything because when I hover over `Olm` in my editor, it says it's of type `any` :) Oh well, good enough for now though.